### ISSUE TITLE
Can't reproduce results on MSRVTT and MSVD dataset

### ISSUE THREAD

**Issue by 1999Lyd** [PKU-YuanGroup/Video-LLaVA#191]
Hi,
I followed the instructions in the TRAIN_AND_VALIDATE.md file, downloaded the dataset, and ran the evaluation script. However, I only achieved 46% accuracy on MSRVTT and 60% accuracy on MSVD, which are both lower than the results reported in the paper. The checkpoint I used is Video-LLaVA-7B.
I’m wondering if there are any additional steps or updated scripts that are not reflected in the README that I should be aware of.
Thank you for your help in advance.

**Comment by wren93** [comment]
Hi, did you figure this out? I got the same result on MSVD

**Comment by 1999Lyd** [comment]
> Hi, did you figure this out? I got the same result on MSVD

I have found the cause, the input arguments ‘do sample’ should be False when temperature is 0, but it is set to be True in the scripts. I hardcoded the temperature above 0 inside transformers script. Btw, my results are still a little bit lower than the result reported in the paper(about 1-2 points) and results for TGIF dataset is much lower than the reported results( only 43%)

**Comment by Jingchensun** [comment]
The same as me, I set the ‘do sample’ as False and the temperature is 0, only get the 36.2 Accuracy on MSVD dataset. Can the author provide some instrutions?

**Comment by Yang-Liz** [comment]
Hi bro, is your training dataset downloaded from the hugging face link provided by the author? If I use this dataset for training, some videos will be missing, and the metrics will be far lower than those reported by the author
